src/cv/insert_frame.py

```diff
@@ -0,0 +1,22 @@
+import cv2
+from datetime import datetime
+import imutils
+import json
+import numpy as np
+import os
+import time
+import settings
+from cassandra.cluster import Cluster
+from cassandra.query import tuple_factory
+
+cluster = Cluster()
+session = cluster.connect()
+session.row_factory = tuple_factory
+
+def insert_frame(camera_id, stream_id, frame_id, frame, objs):
```

Comment:
Where is this used in this pull request?

Reply:
Used in apply_detections.py